packages/atomic/src/components/common/generated-answer/generated-answer-common.tsx

```diff
@@ -203,6 +203,16 @@ export class GeneratedAnswerCommon {
     this.props.getGeneratedAnswer()?.dislike();
   };
 
+  private onChangeAnswerStyle = (answerStyle: GeneratedAnswerStyle) => {
```

Comment:
```suggestion
  private onChangeAnswerStyle(answerStyle: GeneratedAnswerStyle) {
```

Reply:
And could you modify the other methods in this file so they follow the same format ? Thank you !